dashboard/src/components/views/EnvironmentView/EventDetailsContent/EventDetailsContent.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import PropTypes from "prop-types";
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+import Grid from "material-ui/Grid";
+
+import AppContent from "../../../AppContent";
+import Content from "../../../Content";
+import NotFoundView from "../../NotFoundView";
+import EventDetailsCheckResult from "./EventDetailsCheckResult";
+import EventDetailsRelatedEntities from "./EventDetailsRelatedEntities";
+import EventDetailsConfiguration from "./EventDetailsConfiguration";
+import Loader from "../../../Loader";
+
+const query = gql`
+  query EventDetailsContentQuery(
+    $ns: NamespaceInput!
+    $check: String!
+    $entity: String!
+  ) {
+    event(ns: $ns, entity: $entity, check: $check) {
+      id
+      timestamp
+
+      check {
+        ...EventDetailsCheckResult_check
+        ...EventDetailsConfiguration_check
+      }
+      entity {
+        ...EventDetailsCheckResult_entity
+        ...EventDetailsRelatedEntities_entity
+        ...EventDetailsConfiguration_entity
+      }
+    }
+  }
+
+  ${EventDetailsCheckResult.fragments.check}
+  ${EventDetailsCheckResult.fragments.entity}
+  ${EventDetailsRelatedEntities.fragments.entity}
+  ${EventDetailsConfiguration.fragments.check}
+  ${EventDetailsConfiguration.fragments.entity}
+`;
+
+class EventDetailsContent extends React.PureComponent {
+  static propTypes = {
+    match: PropTypes.object.isRequired,
+  };
+
+  render() {
+    const { match } = this.props;
+    const ns = {
+      organization: match.params.organization,
+      environment: match.params.environment,
+    };
+
+    return (
+      <Query query={query} variables={{ ...match.params, ns }}>
```

Comment:
Set the `fetchPolicy` to `cache-and-network` so the user will receive updated data every time the view is mounted.